daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -670,6 +670,12 @@ private[lf] object Speedy {
     // Raises an exception if missing a package.
     private[speedy] def importValue(typ0: Type, value0: V[V.ContractId]): Unit = {
 
+      def assertRight[X](x: Either[LookupError, X]) =
+        x match {
+          case Right(value) => value
+          case Left(error) => crash(error.pretty)
```

Comment:
similar question here. I think a comment on why we always want to crash for this is useful.

Reply:
Same answer.

Reply:
20b2bb95da79a10d962f2fe57d22cb08f5825bc3